tests/e2e/scale/test_delete_local_volume.py

```diff
@@ -0,0 +1,68 @@
+import logging
```

Comment:
This test is developed as part of closed loop process.
Do you think it is a good idea to have a separate directory for all closed loop bugs? or we move this to the existing directories wherever it suites
@hnallurv @ebenahar @vasukulkarni Please share your thoughts

Reply:
I don't think we need a dedicated dir for this effort. Each test should be added to the relevant place